### PR TITLE
feat(Table): set new basis Table styles including medium and small size

### PR DESCRIPTION
**Pull request by tujoworker**
The new styles ([preview](https://eufemia-feattablestyles.gatsbyjs.io/uilib/components/table/)):

- Font-color: `black-80` 
- Sort button color: `sea-green` 
- "Even" row background color: `black-3` 
- Border and outline color: `black-8` 
- Three different sizes: large, medium and small

Use fixed sizing instead of `var(--spacing-...)` because the "component" supports its own sizes now.

New

Screenshot 2022-11-22 at 15 03 19

Old

Screenshot 2022-11-22 at 15 04 15

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ showTabs: true
...
 | `stickyOffset`                              | _(optional)_ defines the offset (top) in `rem` from where the header should start to stick. You may define your app header height here, if you have a sticky header on your page. Defaults to `0`.   |
 | ~~`variant`~~ (not implemented yet)         | _(coming)_ defines the style variant of the table. Options: `basis` . Defaults to `generic`.                                                                                                         |
-| ~~`size`~~ (not implemented yet)            | _(coming)_ spacing size inside the table header and data. Options: `small` \| `medium` \| `large` \. Defaults to `large`.                                                                            |
+| `size`                                      | _(coming)_ spacing size inside the table header and data. Options: `small` \| `medium` \| `large` \. Defaults to `large`.                                                                            |
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md:16]:
```suggestion
| `size`                                      | _(optional)_ spacing size inside the table header and data. Options: `small` \| `medium` \| `large` \. Defaults to `large`.                                                                            |
```

**Review by langz — APPROVED** [review]
Looks nice 👨‍🎨 🎨

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md:16]: Ohh, yes 👍 💯

**Review by NancyBolstad — APPROVED** [review]
Nice, thanks! Looking forward to implement table styles in dnb-web 👍

**Pull request merged**